src/components/BrowserNavigation/BrowserNavigation.test.tsx

```diff
@@ -48,6 +48,10 @@ test('allows user to create a new query', () => {
 
   expect(store.getActions()).toMatchInlineSnapshot(`
     Array [
+      Object {
```

Comment:
I checked `createNewQuery` saga and `resetSavedQuery` is part of saga flow.
Do we need to additionally perform `dispatch(createNewQuery());` in BrowserNavigation component ?

```
export function* createNewQuery() {
  yield put(setViewMode('editor'));
  const pubsub = yield getContext(PUBSUB_CONTEXT);
  yield pubsub.publish(NEW_QUERY_EVENT);

  yield put(queriesActions.resetQueryResults());
  yield put(resetVisualization());
  yield put(savedQueryActions.resetSavedQuery());
}
```